### ISSUE TITLE
Can not detect Kinect Model 1414 on Windows 8.1

### ISSUE THREAD

**Issue by XOIIO** [shiffman/OpenKinect-for-Processing#41]
I recall using this quite some time ago on windows 8.1, however since the update, the files I had stopped working, and with a fresh install of processing, and downloads of the library it refuses to detect my kinect. I am trying on processing 2.2.1 and on 3, but it fails to detect the kinect. Is there an updated driver I need which I missed? I scoured the page and couldn't see any mention of it.

**Comment by ThomasLengeling** [comment]
for windows you need to install the USB drivers that are mention in the requirements

**Comment by XOIIO** [comment]
Tried that, and replaced the drivers for the everything that seems related to the kinect (on it's port), I don't have a WCID on any of those, and not sure if it is supposed to show up all as one or not,
![screenshot 104](https://cloud.githubusercontent.com/assets/12793421/10407865/1d4f6b42-6eae-11e5-80c0-9d05a6e873c4.jpg)

**Comment by XOIIO** [comment]
Made some progress after fiddling back and forth, there was a kinect management service running, after I got rid of that and removed and re-plugged the kinect it works, for the motor at least, however I can't get any video feeds, and it doesn't list them in zadig either. Seems that if you had the sdk running that can cause an issue perhaps though

**Comment by XOIIO** [comment]
Alright so, got camera working with it, however when switching back to the windows driver, the developer sdk can't see it anymore, interesting.

**Comment by ThomasLengeling** [comment]
great, to switch back to the SDK, you need to do

```
To uninstall the libusbK driver (and get back the official SDK driver, if installed):

Open Device Manager
Under libusbK USB Devices, right click the "Xbox NUI Sensor (Composite Parent)" device and select uninstall.
Important: Check the "Delete the driver software for this device." checkbox, then click OK.
```

**Comment by XOIIO** [comment]
Tried nuking out the drivers and reinstalling them from the SDK several times, I got kinect for windows security to pop back up as well as "audios" in computer management but the sdk refuses to see it still, as well as other software. They showed up in zadig a couple times but now do not again
![screenshot 105](https://cloud.githubusercontent.com/assets/12793421/10410829/f8c65d6e-6f0a-11e5-8e30-da40893ca954.jpg)

Going to re-download the whole sdk to try that.

**Comment by XOIIO** [comment]
Still no luck, and I can't see "kinect for windows security" anymore, I can get it working with processing just fine but I can't get anything using the windows sdk to recognize it.

**Comment by XOIIO** [comment]
Still refusing to work, I don't get it. Maybe it's because the driver source is libusb? Problem is I have tired uninstalling all the drivers, uninstalling the kinect for windows drivers from control panel, and then reinstalled them with the sdk from microsoft but they still have this version, I recall the ones listed in zadig were newer than what I had, but I've nuked everything out and the damn thing still won't work.

I wish I had checked these before so I could compare them and see if there's some sort of issue. I can't find the kinect drivers anywhere in system32 to make sure they are all gone, so I don't know what the heck I can do.
![screenshot 106](https://cloud.githubusercontent.com/assets/12793421/10411343/d53e0090-6f1e-11e5-8522-97790d74ac48.jpg)

**Comment by XOIIO** [comment]
The only thing I can think of is if it's because of the drivers being from libusb, but I can't get rid of them no matter what I try, and I can't find where they are actually installed to, so maybe that would do it if someone can point me there.

**Comment by XOIIO** [comment]
Booted up my old laptop and found that with just the sdk it seems all the kinect devices are under "kinect for windows", not usb devices, it must be the libusb drivers causing this, but I can't find out how to get rid of them. I mean I nuked out anything saying kinect or libusb, so what the heck :/
![screenshot 32](https://cloud.githubusercontent.com/assets/12793421/10411552/05061c42-6f27-11e5-8816-cc611dffd03b.jpg)

**Comment by XOIIO** [comment]
Starting to go through the registry, some stuff was pointing to a folder called usb_driver under my username directory, deleted that but it still didn't work, and for some reason I can't delete these registry entries.Uuuuhg

**Comment by XOIIO** [comment]
God, finally, I got a headache over this. I had to use rootkit tools and runassystem to delete registry keys for this, which didn't do the trick, I also tracked down the libusb ones, but then "kinect for windows camera" wouldn't work, apparently that one got wiped out too, so I copied that registry value after finally tracking the damn thing down, edited it to match the others from my main computer which didn't work, so I had to nuke out even more kinect entries, and then try and find a registry fix tool which actually worked with windows 8, since windows 8 doesn't have scanreg anymore.

Finally got the f**ing thing working though, god it was such a massive hassle.
